config.js

```diff
@@ -0,0 +1,29 @@
+var chalk = require('chalk');
+
+function getChalkColors(configInput) {
+  // make the read file understandable to chalk.
+  var chalkColors = {};
+  for (var color in configInput.colors) {
+  	if (configInput.colors.hasOwnProperty(color)) {
+  		var colorFunc = chalk;
+      //build up the chalk function
+      configInput.colors[color].forEach(function(style) {
+  			colorFunc = colorFunc[style];
+  		});
+  		chalkColors[color] = colorFunc;
+  	}
+  }
+  return chalkColors;
+}
+
+var configInput = require("./lmeDefaultConfig.json");
+try {
+	// try to get the config file from root file of the application.
+	var providedConfig = require('app-root-path').require("lmeconfig.json");
+  configInput.colors = Object.assign({}, configInput.colors, providedConfig.colors);

```

Comment:
Yeah,  otherwise you will not get your desired behaviour.  I am sure there are other ways of doing this,  but using object. Assign you need to specify... Go try it out in a sandbox project ;)